announcements/2020-Week-29/kvm.md

```diff
@@ -0,0 +1,4 @@
+# :zap: This Week in KVM Releases (July 17, 2020):zap:
+
+
+Please let <!subteam^S0GSG846L> know if you have any feedback or questions by replying to this announcement in a thread.
```

Comment:
```suggestion
Please reply to this announcement in a thread if you have any feedback or questions.
```
PO group handle doesn't look good, remove for now?

Reply:
sure!